src/goblint_lib.ml

```diff
@@ -92,6 +92,7 @@ module Malloc_null = Malloc_null
 module MemLeak = MemLeak
 module UseAfterFree = UseAfterFree
 module MemOutOfBounds = MemOutOfBounds
+module C2poAnalysis = C2poAnalysis
```

Comment:
This is better suited for the "Value" subsection where all other relational analyses are already. This analysis also handles pointers to locals/globals, no?